lib/AArch64Base.ml

```diff
@@ -1709,6 +1709,64 @@ type 'k kinstruction =
   | I_STZ2G of reg * reg * 'k idx
   | I_LDG of reg * reg * 'k
   | I_UDF of 'k
+(* Pointer Authentication Code: Basic instruction only
+   See C3.1.10 for the complete list of instructions
```

Comment:
Not sure section numbers in the Arm ARM are stable. For instance, in L.a. "C6.2.266" is something else, so it must have changed since you wrote this comment.

Perhaps, mentioning the issue could help. For instance, "Arm ARM Issue K.a, C3.1.10".